server/controllers/UserController.js

```diff
@@ -10,18 +10,18 @@ const User = {
    * @returns {object} user object
    */
   create(req, res) {
-    const { firstname, lastname, email, password, username } = req.body;
-    if (!firstname || !lastname || !email || !password) {
+    const { firstName, lastName, email, password, userName } = req.body;
+    if (!firstName || !lastName || !email || !password) {
       return res.status(400).send({ message: 'All fields are required' });
     }
     const newUser = {
       id: uuid.v4(),
-      firstname,
-      lastname,
+      firstName,
+      lastName,
       email,
       createdDate: moment.now(),
       password,
-      username: username || firstname + lastname,
+      userName: userName || `${firstName}${lastName}`,
```

Comment:
There should be a space between `${firstName}` and `${lastName}` unless you don't want a space between `firstName` and `lastName`